src/main/java/org/kohsuke/github/GHTeam.java

```diff
@@ -151,6 +152,19 @@ public PagedIterable<GHUser> listMembers(String role) throws IOException {
         return root().createRequest().withUrlPath(api("/members")).with("role", role).toIterable(GHUser[].class, null);
     }
 
+    /**
+     * List members with specified role paged iterable.
+     *
+     * @param role
+     *            the role
+     * @return the paged iterable
+     * @throws IOException
+     *             the io exception
+     */
+    public PagedIterable<GHUser> listMembers(Role role) throws IOException {
+        return listMembers(role.name().toLowerCase(Locale.ROOT));
+    }
```

Comment:
Added this in passing as I wanted to test this and it's more practical with the enum rather than the string.

Reply:
Modifed to use `transformEnum()`, but otherwise 👍 .